\documentclass[12pt]{paper}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{color}
\begin{document}
\title{Definitions-Biology}
\maketitle
\section{Histones}
...

Hyperaccetlation of the histones leads to unfolding of the chromatin that should facilitate the general accessibility of factors to the DNA \cite{Blackwood98}.

\section{Chromatin remodeling machines}
Many of the multi-subunit molecular machines include proteins from yeasts (SWI/SNF, RSC), Drosophila (ACF, BRM), and mammalian (BRG1) \cite{kadonaga1998eukaryotic}. These are members of the NTP binding proteins. It has been postulated that the NTP-binding subunit might act as an ATP driven DNA translocating motor that disrupts DNA-histone interactions. 

The core histones (H3 and H4) can be acetylated in the N-terminal tails which was postulated to reduce the affinity of the histone to the bound DNA, and allow more access to transcription factors. The acytilation causes charge neutralization, in the histones \cite{kadonaga1998eukaryotic}. there is a correlation (not universal) between histone acetylation and gene activity. 

Two major classes of protein play a role in allowing access to the DNA: (1) Covalent histone-modifyin complexes, and (2) ATP-dependent chromatin remodeling complexes.
The covalent histone modifying complexes modify the histone affinity by methylation phosphorylation, etc.. in the N-terminal of histones. These modification change the affinity between histones and DNA, either tightening the bond or loosening it. 

\section{Enhancers}
 An \href{http://en.wikipedia.org/wiki/Enhancer_(genetics)}{Enhancer} is a short (20-400bp,\cite{Kulaeva12}) region of DNA that can be bound with proteins to activate transcription of a gene or genes. These proteins are usually referred to as transcription factors. Enhancers contain  specific sequence motifs with which DNA-binding proteins interact and transmit molecular signals to genes \cite{Blackwood98}.

Enhancers increase transcription of genes in a manner that is independent of their orientation and distance relative to the RNA start site \cite{Blackwood98} 


\section{Promoters}
Promoters (or core promoters) are located within $\pm 40$ nucleotides from the RNA start site \cite{Blackwood98}. Promoters are identified according to their TATA box. 


\section{TATA box}
Considered to be the core promoter region, it is a short sequence of DNA and, which acts as the binding site for general transcription factors. TATA box contains the sequence 5'-TATAAA-3' . It is usually located within 25 bp of the transcription start site. About 24$\%$ of the genome contains TATA box sites. In the process of transcription, the TATA binding protein binds to the TATA box an unwinds the DNA. Because the TATA box is rich with AT connections, it facilitates easier unwinding of the DNA, due to the weaker base-pair interactions). Most genes lack TATA box and use instead an initiator element. Only about 10$\%$ of the promoters in humans depend on TATA boxes. 

\section{Insulators}

\section{Promoter-Enhancer Interactions}
In the majority of cases, action of enhancers involves enhancer-promoter interaction through proteins bound at the enhancer and promoter, accompanied by formation of an intervening chromatin loop \cite{Kulaeva12}.

There are two mechanisms by which enhancer-promoter  selectivity might be achieved. First, there could be specific interactions between enhancer -binding proteins and factors that interact with the promoter. Second, boundary elements (insulators) could be used to block undesired enhancer-promoter interactions \cite{Blackwood98}. 

How might enhancer binding proteins and their associated co-activators  establish productive interaction with the promoter? One option is the DNA looping. A "facilitated tracking" mechanism for enhancer function is postulated to allow enhancer bound complex containing DNA-binding factors and co-activators "tracks" via small steps along the chromatin until it encounters the promoter, at which a stable looped structure is formed \cite{Blackwood98}. After formation of the activation chromatin loop, it is likely stabilized by additional protein factors, such as CTCF and cohesin \cite{Kulaeva12}. CTCF is an insulator DNA binding protein. 

It was shown that formation of chromatin loop topologically isolating the enhancer from the target promoter is sufficient to block enhancer-promoter communication. 


For an illustration of the participants in gene transcription see Figure \ref{mechanismOfGeneTranscription}

\begin{figure}[H]
\includegraphics[scale=0.5]{mechanismOfGeneTransciption}
\caption{The participants in the process of gene transcription. The enhancer is connected to the mediator protein which recruits RNA polymerase to the process. The mediator gene is connected to the activator protein, which is connected to promoter proximal elements. These elements are located within $\sim$40 bp of the promoter, a property which allows the RNA polymerase to quickly find the promoter and to initiate transcription}
\label{mechanismOfGeneTranscription}
\end{figure}

\section{CTCF}\label{section_CTCF}

\section{Primers}\label{section_primers}
primer design in the process of strand amplification, such as in the polymerase chain reaction (PCR) involves \textit{forward} and \textit{reverse} primers. The completion of a single strand DNA (or RNA) by polymerase can only be made on an already existing complementary segment. The convention is to write the genetic sequence from the 5' end to the 3' end, so that the filling-in is done to elongate the 3' end of the top single strand on top of the bottom strand (from 3' to 5')
% The bibliography
\bibliographystyle{plain}
\bibliography{BiologicalDefinitionsBibliography} % the bibliography.bib file 
\end{document}